database/seeds/CamelSeeder.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class CamelSeeder extends Seeder
```

Comment:
For me the seeder fails with the error:
`Target class [CamelSeeder] does not exist.`

Reply:
Managed to solve this at my side, I had to run `composer dump-autoload`